docs/book/src/migration/v2vsv3.md

```diff
@@ -2,21 +2,90 @@
 
 This document covers all breaking changes when migrating from v2 to v3.
 
-The details of all changes (breaking or otherwise) can be found in [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases) release notes.
+The details of all changes (breaking or otherwise) can be found in
+[controller-runtime][controller-runtime],
+[controller-tools][controller-tools]
+and [kb-releases] release notes.
+
+## Common changes
+
+v3 projects use Go modules and request Go 1.15+. Dep is no longer supported for dependency management.
 
 ## Kubebuilder
 
-- A plugin design was introduced to the project. For more info see the [Extensible CLI and Scaffolding Plugins][plugins-phase1-design-doc]
-- The GO supported version was upgraded from 1.13+ to 1.15+
+- Preliminary support for plugins was added. For more info see the [Extensible CLI and Scaffolding Plugins][plugins-phase1-design-doc].
+
+- The `PROJECT` file now has a new layout.  It stores more information about what resources are in use, to better enable plugins to make useful decisions when scaffolding.
+    
+    Furthermore, the PROJECT file itself is now versioned: the `version` field corresponds to the version of the PROJECT file itself, while the `layout` field indicates the scaffolding & primary plugin version in use. 
+    
+- Upgrade of the version `gcr.io/kubebuilder/kube-rbac-proxy`, which is an optional component enabled by default to secure the request made against the manager, from `0.5.0` to `0.8.0` to address security concerns. The details of all changes can be found in [kube-rbac-proxy][kube-rbac-proxy].
```

Comment:
i dont understand this line. Is it instructing the reader to MAKE this change or telling the reader that this change was made for them or telling the reader that this is some version that was changed in the background but they don't have to directly worry about?

Reply:
I got your point here. re-phased